Makefile

```diff
@@ -71,6 +71,14 @@ build-all:
 	go build -o receptor --tags no_controlsvc,no_backends,no_services,no_tls_config,no_workceptor,no_cert_auth ./cmd/receptor-cl && \
 	go build -o receptor ./cmd/receptor-cl
 
+DIST := receptor_$(shell echo '$(VERSION)' | sed 's/^v//')_$(GOOS)_$(GOARCH)
+build-package:
```

Comment:
Looks really good, 
Can you add cleaning the `dist` dir to the `clean:` make target please?

Reply:
Thanks, I totally forgot about `clean`. Added.